src/topotoolbox/utils.py

```diff
@@ -103,7 +103,7 @@ def read_tif(path: str) -> GridObject:
         grid.cellsize = dataset.res[0]
         grid.bounds = dataset.bounds
         grid.transform = dataset.transform
-        grid.crs = dataset.crs
+        grid.georef = dataset.georef
```

Comment:
This should be

```python
grid.georef = dataset.crs
```

The `dataset` comes from rasterio, so it calls its coordinate information `crs`.